common/djangoapps/util/tests/test_db.py

```diff
@@ -235,10 +234,7 @@ def test_migrations_are_in_sync(self):
         make a migrationless release that'll require a separate migration
         release afterwards, this test doesn't fail.
         """
-        # TODO: Remove Django 1.11 upgrade shim
-        # SHIM: Migrations are known to be needed when actually bumping the Django version number.
-        if django.VERSION >= (1, 9):
-            pytest.skip("Migrations are known to be needed for Django 1.9+!")
+        pytest.skip("Migrations are known to be needed for Django 1.9+!")
```

Comment:
It's weird to have a test in the codebase that's always skipped. And the skip message is far from clear. Did we skip this test because it took so long in Django 1.8? I feel like this test should either be deleted or run all the time now,

Reply:
Oh hey I was going to ask about that too. IIRC this is actually a Very Important Test that makes sure people don't submit PRs that need migrations, but forget to add them. I think we want to re-enable, but @jmbowman probably knows for sure.

Reply:
From Jeremy's PR marking this as a known failure: "Mark the migration sync test as known to fail in the newer Django versions until we release new migrations with the Django 1.11 upgrade." Sounds like a re-enable to me.

Reply:
Jeremy confirmed, adding it back in now!